### PR TITLE
Aerial menu

### PR DESCRIPTION
**Pull request by PhucCes**
https://app.asana.com/0/596117517607240/602587710995373/f

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "gulp-sourcemaps": "^2.6.*",
     "gulp-standard": "^10.1.2",
     "gulp-stylelint": "^6.0.0",
-    "gulp-uglify": "^3.0.*",
+    "gulp-uglify-es": "^1.0.1",
```
Comment (alexgurrola) [inline package.json]: That's a nice addition.

File: directives/edit.js [hunk 2]
```diff
@@ -157,7 +157,7 @@
 
           // WATCHERS
 
-          $scope.$watch('model.data.' + $scope.property, function (data) {
+          $scope.$watch('model.' + $scope.property, function (data) {
```
Comment (alexgurrola) [inline directives/edit.js]: I wouldn't hoist the watcher to the root model.  What do you need to watch up there?

**Comment by PhucCes** [comment]
Please check the Travis errors. I believe it not mine, it belong to our syntax, I think you may need to further config on the Travis.

**Comment by alexgurrola** [comment]
Travis dies because `MenuLink.js` doesn't pass the syntax of `Standard.js`.  I fixed `env.js` earlier today.

**Pull request merged**